fastlane/Fastfile

```diff
@@ -115,10 +115,24 @@ platform :ios do
         build_and_upload_app(options)
     end
 
+    desc "Build & Upload Col 2 Restricted AppStore build"
+    lane :appstore_col_2_restricted do |options|
```

Comment:
Looks good, could you already make some test runs from this branch to ensure it works as expected? Or will we do it from `develop` then?